backend/battling/views.py

```diff
@@ -24,16 +20,18 @@ class CreateBattle(CreateView):
 
     def form_valid(self, form):
         form.instance.creator = self.request.user
+        form.instance.status = "ONGOING"
         battle = form.save()
 
         team_creator = Team.objects.create(battle=battle, trainer=self.request.user)
 
-        team_opponent = Team.objects.create(battle=battle, trainer=battle.opponent)
-
-        send_battle_invite(battle, team_opponent.id)
```

Comment:
You removed this. Is that right?

Reply:
Yes... I removed it so it would work on Heroku, because it errors when I send emails.

Reply:
OK!